src/components/deploy-table.tag

```diff
@@ -0,0 +1,31 @@
+<DeployTable>
+
+  Service: <input name="service" onkeyup={ edit } placeholder="Try Media"/>
+  <table class="table">
+    <thead>
+      <tr>
+        <th>Name</th>
+        <th>Build Time</th>
+        <th>Status</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr each="{ item in list }">
+        <td>{ item.service }</td>
+        <td>{ item.minutes } minutes(s)</td>
+        <td>{ item.success ? 'Success' : 'Fail' }</td>
+      </tr>
+    </tbody>
+  </table>
+
+  <script type="es6">
+
+    const self = this
+    self.list = self.deploys.list
+
+    self.edit = function(e) {
+      self.list = self.deploys.search(e.target.value)
+    }
```

Comment:
Since this is using ES6 we can cut some of the code down by switching to an arrow function:

```
this.list = this.deploys.list
this.edit = e => {
  this.list = this.deploys.search(e.target.value)
}
```